andela/travel_tool_back
# #160833565 Notifications for rejected requests

#### What does this PR do?
* This PR sends notifications for rejected requests
#### Description of Task to be completed?
* Update the `sendNotificationApproval` method to check if request was rejected and send appropriate  message

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - cd travel_tool_back
- Rollback and run database migrations - yarn db:rollmigrate
- Install dependencies and start the server - make start
-  [Start](https://github.com/andela/travel_tool_front/pull/107)  the frontend.
- Make a request and choose yourself as the manager
- `Reject` the request
- You should see a `rejection` notification message in the `general nofitication pane`
#### Any background context you want to provide?
- In other to test this feature, you need to update your role on the Users table to that of a manager.
Manager roleId: `50319`

#### What are the relevant pivotal tracker stories?
* [#160833565](https://www.pivotaltracker.com/story/show/160833565)

#### Screenshots (if appropriate)
* N/A
#### Questions:
* N/A
